### PR TITLE
Update journal slots with new layout and styling

### PR DESCRIPTION
**Pull request by buzzub**
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [ ] Code has been tested manually
- [ ] Tests are passing
- [ ] PR link added to JIRA
- [ ] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

### CODE REVIEW

File: src/pages/journal/components/indicators/indicators.scss [hunk 1]
```diff
@@ -1,8 +1,6 @@
 indicators {
   div {
-    display: flex;
-    justify-content: center;
-    width: 2em;
+    padding-left: 8px;
```
Comment (ammarhaiderbjss) [inline src/pages/journal/components/indicators/indicators.scss]: Is there a case for storing these values in sass variables? We'll probably have this same amount of padding in various places so putting it in a variable might make sense.

Comment (buzzub) [reply src/pages/journal/components/indicators/indicators.scss]: Good shout. In this instance I can see how I maybe could unify the layout on the welsh logo above and indicator below. Currently they both have 16px total padding. The welsh logo has 4 + 6 + 6, and the indicator icon has 4 + 4 + 4 + 8.

File: src/theme/sass-partials/_buttons.scss [hunk 2]
```diff
@@ -2,10 +2,34 @@
   background: map-get($colors-gds, gds-dark-green);
   box-shadow: inset 0 -2px 0 0 map-get($colors-gds, gds-green);
   border-radius: 10px;
+  margin: 0 0.2rem;
+  h3 {
+    color: map-get($colors, mes-white);;
+  }
```
Comment (Cavalier07) [inline src/theme/sass-partials/_buttons.scss]: Can we remove the double ; theres a few in this file

File: src/pages/journal/components/test-outcome/test-outcome.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
 test-outcome {
-  
+  .button-ios {
+    margin: 0px;
+  }
```
Comment (Cavalier07) [inline src/pages/journal/components/test-outcome/test-outcome.scss]: shouldn't need the px :)

**Comment by ammarhaiderbjss** [comment]
@buzzub Could we get a screenshot of the lovely new journal please?

**Comment by buzzub** [comment]
simulator screen shot - ipad pro 10 5-inch - 2019-02-14 at 15 56 59

simulator screen shot - ipad pro 10 5-inch - 2019-02-14 at 15 56 42

**Comment by buzzub** [comment]
@ammarhaiderbjss Screenshots added

File: src/pages/journal/components/test-slot/test-slot.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { SlotComponent } from '../slot/slot';
 import { ScreenOrientation } from '@ionic-native/screen-orientation';
 import { vehicleDetails } from './test-slot.constants';
 import { TestCategory } from '../../../../common/test-category';
+// import { PassFinalisationPageModule } from '../../../pass-finalisation/pass-finalisation.module';
```
Comment (ammarhaiderbjss) [inline src/pages/journal/components/test-slot/test-slot.ts]: Can we delete this if not needed?

File: src/pages/journal/journal.ts [hunk 5]
```diff
@@ -173,11 +173,15 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
...
     const slots = this.slotSelector.getSlotTypes(emission);
+    let lastLocation;
     for (const slot of slots) {
       const factory = this.resolver.resolveComponentFactory(slot.component);
       const componentRef = this.slotContainer.createComponent(factory);
       (<SlotComponent>componentRef.instance).slot = slot.slotData;
       (<SlotComponent>componentRef.instance).hasSlotChanged = slot.hasSlotChanged;
+      (<SlotComponent>componentRef.instance).showLocation = (slot.slotData.testCentre.centreName !== lastLocation);
+      lastLocation = slot.slotData.testCentre.centreName;
+      console.log(slot.slotData);
```
Comment (ammarhaiderbjss) [inline src/pages/journal/journal.ts]: Remove console log?

Comment (buzzub) [reply src/pages/journal/journal.ts]: Good spot

File: src/pages/journal/components/test-slot/test-slot.ts (hunk 4 above)
Comment (buzzub) [reply src/pages/journal/components/test-slot/test-slot.ts]: Done

**Pull request merged**